css-two-column-layout/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container ">
+    <div class="row">
+      <div class="column-full">
+        <h1 class="bighead"> Scale, Speed, Agility</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-full second-row">
+        <p> We've dedicated thousands of man hours to making healthcare instituitions in the Helium
+          newtowrk run 10x more efficiently and deliver an unparalleled quality of care. </p>
+      </div>
+    </div>
+    <div class="row bigrow">
```

Comment:
The class name `bigrow` here gives very little insight into its purpose, you should try to make a more descriptive class name here. Additionally, class names that include multiple words should have the words separated by dashes, which is known as `kebab-case`.